server/easyrtc-server.js

```diff
@@ -8,8 +8,8 @@ const easyrtc = require("open-easyrtc");      // EasyRTC external module
 // Set process name
 process.title = "networked-aframe-server";
 
-// Get port or default to 8080
-const port = process.env.PORT || 8080;
+// Get port or default to 5832
+const port = process.env.PORT || 5832;

```

Comment:
Please don't change the default port in the PR.

Reply:
If you need to change the port, you can do this in bash in your terminal:

```
export PORT=5832
npm start
```

and even put this line in your `~/.bashrc`
so next time you just use `npm start` and this will use the redefined port.

Reply:
ok thanks, I didn't know that way !